pkg/operation/botanist/controlplane.go

```diff
@@ -17,9 +17,11 @@ package botanist
 import (
 	"context"
 	"fmt"
+	"hash/crc32"
 	"path/filepath"
 	"time"
 
+	druidv1alpha1 "github.com/gardener/etcd-druid/api/v1alpha1"
```

Comment:
I think the convention in g/g is to first place the golang imports, then the ones from the same project, and then the external ones. `druidv1alpha1 "github.com/gardener/etcd-druid/api/v1alpha1"` needs to go in the group below.

Reply:
Done